doc/classes/CameraServer.xml

```diff
@@ -37,13 +37,27 @@
 				Returns the number of [CameraFeed]s registered.
 			</description>
 		</method>
+		<method name="is_monitoring_feeds" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns `true` if the server is actively monitoring camera feeds.
```

Comment:
```suggestion
				Returns [code]true[/code] if the server is actively monitoring camera feeds.
```

Reply:
This wasn't resolved.

Reply:
Weird, I made the change locally but it's not online. Will probably get resolved on the next push :)